src/mxe-conf.mk

```diff
@@ -26,10 +26,10 @@ define $(PKG)_BUILD
     touch '$(CMAKE_TOOLCHAIN_DIR)/.gitkeep'
     (echo 'set(CMAKE_SYSTEM_NAME Windows)'; \
      echo 'set(MSYS 1)'; \
-     echo 'set(BUILD_SHARED_LIBS $(CMAKE_SHARED_BOOL))'; \
-     echo 'set(BUILD_STATIC_LIBS $(CMAKE_STATIC_BOOL))'; \
-     echo 'set(BUILD_SHARED $(CMAKE_SHARED_BOOL))'; \
-     echo 'set(BUILD_STATIC $(CMAKE_STATIC_BOOL))'; \
+     echo 'set(BUILD_SHARED_LIBS $(CMAKE_SHARED_BOOL) CACHE BOOL "BUILD_SHARED_LIBS" FORCE)'; \
+     echo 'set(BUILD_STATIC_LIBS $(CMAKE_STATIC_BOOL) CACHE BOOL "BUILD_STATIC_LIBS" FORCE)'; \
+     echo 'set(BUILD_SHARED $(CMAKE_SHARED_BOOL) CACHE BOOL "BUILD_SHARED" FORCE)'; \
+     echo 'set(BUILD_STATIC $(CMAKE_STATIC_BOOL) CACHE BOOL "BUILD_STATIC" FORCE)'; \
```

Comment:
Can you elaborate on this, please? Do you need to redefine BUILD_SHARED_LIBS in a CMakeLists.txt of a library?